src/modules/auth/services/auth-service/auth.service.ts

```diff
@@ -22,15 +23,28 @@ export class AuthService {
 	): Promise<void> {
 		const { redirectTo, slug, ...otherQueryParams } = query;
 		let originalPath: string = (redirectTo as string) || router.asPath || '';
+
+		// Don't redirect the user back to logout, after they logged in
 		if ((originalPath || '').endsWith('/' + ROUTE_PARTS.logout)) {
 			originalPath = '/';
 		}
+
+		// Redirect /slug to the search page with filter
 		if (slug) {
-			originalPath = `/${ROUTE_PARTS.search}/${slug}`;
+			originalPath = `/${ROUTE_PARTS.search}?${ROUTE_PARTS.maintainer}=${slug}`;
```

Comment:
I'd use the same enum as used on the filter page to keep them in sync instead of adding those qp to the routes: `VisitorSpaceFilterId.Maintainer`

Reply:
I'd like to start pulling them apart since VisitorSpaceFilterId is used as the field names in the backend for the api
And ROUTE_PARTS is used in the frontend for everything related to routes. These are also often in dutch. While the api field names will always be in english.

I think we'll have to do that anyways, since meemoo wants to see: `aanbieder=huis-van-alijn` in the url and not `maintainer=huis-van-alijn`

Reply:
ok, i added a todo to change it
and revert back to `VisitorSpaceFilterId.Maintainer` for now